src/patternfly/components/Login/docs/code.md

```diff
@@ -12,11 +12,12 @@
 | `.pf-c-login__main-header .pf-c-dropdown` | `<div>` |  Creates a language selector dropdown in the main header area. |
 | `.pf-c-login__main-body` | `<div>` |  Creates the body of the main area. |
 | `.pf-c-login__main-body .pf-c-form` | `<form>` |  Creates the login form in the main body area. |
+| `.pf-c-login__main-body .pf-c-form .pf-c-form-helper-text.pf-m-error` | `<form>` |  Creates the error text above the form fields, this should always be present on the page with `.pf-m-hidden` applied when it is not active. |
```

Comment:
Could we change this to:
`Creates the error text above the form fields. **Required**. When not active, apply `.pf-m-hidden`.